# Offset applied to object position as a pre/post gripping position [m]
#! pre gripping position is currently disabled
pre_goal_retraction_x: 0.1
post_goal_retraction_z: 0.08

# Number of retries of each procedure
procedure_retry_threshold: 5

wait_time_move_group_servers: 120.0

move_group_planning_time: 60.0
move_group_planning_attempts: 10
move_group_goal_orientation_tolerance: 0.1
move_group_goal_position_tolerance: 0.01

arm_name: right_arm
base_link_frame: base_footprint
gripper_frame: gripping_right_link
grasping_group_name: hand
home_position_target_name: Home

# 48mm
opened_gripper_value: 0.0041
# 25mm
closed_gripper_value: -0.0033

# Bottle size
bottle_cap_height: 0.03
bottle_cap_radius: 0.015

# Lower than real height - otherwise it colides with table scene box and plan
# to pick up bottle isn't good
bottle_height: 0.1
bottle_radius_pre_picking: 0.06
bottle_radius_post_picking: 0.03
